docs/guide/howto.md

```diff
@@ -0,0 +1,234 @@
+## Stratis how-to/walk-through ##
+
+
+### Introduction ###
+
+[Stratis](https://github.com/stratis-storage)
```

Comment:
Please omit parenthetical remarks. Either they will become stale, like the one about the D-Bus API, or they are important information, which shouldn't be parenthesized.

Reply:
So you're advocating that we shouldn't use hyperlinks because they can become stale?  That has been true since 1991...

Reply:
I think it makes more sense if this comment is actually about the parenthetical on the next line down.

Reply:
> So you're advocating that we shouldn't use hyperlinks because they can become stale? That has been true since 1991...

No. Just the "parenthetical remarks", exactly as I stated.